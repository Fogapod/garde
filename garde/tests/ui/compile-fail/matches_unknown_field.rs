#![allow(dead_code)]

#[derive(garde::Validate)]
struct Test<'a> {
    #[garde(matches(foo))]
    bar: &'a str,
}

fn main() {}
