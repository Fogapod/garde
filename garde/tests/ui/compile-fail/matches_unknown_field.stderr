error[E0609]: no field `foo` on type `&Test<'a>`
 --> tests/ui/compile-fail/matches_unknown_field.rs
  |
  |     #[garde(matches(foo))]
  |                     ^^^ unknown field
  |
  = note: available field is: `bar`
